app/Dockerfile.redis

```diff
@@ -0,0 +1 @@
+FROM redis:latest
```

Comment:
We prob wanna configure some credentials for the db here instead of leaving them default.